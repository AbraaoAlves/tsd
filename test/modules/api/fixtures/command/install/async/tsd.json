{
  "typingsPath": "D:\\_Editing\\github\\tsd\\tsd-origin\\test\\modules\\api\\tmp\\command\\install\\async\\typings",
  "version": "v4",
  "repo": "borisyankov/DefinitelyTyped",
  "ref": "master",
  "installed": {
    "async/async.d.ts": {
      "commit": "a8d8c2ac668fbb99cb60fb5d25c663e9356fee14",
      "blob": "5ba56e0dbb7cc433d5b98ab3cbbf5e0d92954ff1"
    }
  }
}