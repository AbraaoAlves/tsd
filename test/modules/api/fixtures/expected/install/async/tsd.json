{
  "typingsPath": "D:\\_Editing\\github\\tsd\\tsd-origin\\test\\modules\\api\\tmp\\result\\install\\async\\typings",
  "version": "v4",
  "repo": "borisyankov/DefinitelyTyped",
  "ref": "master",
  "installed": {
    "async/async.d.ts": {
      "commit": "3bc329f49a971b3dae92ca682cb81816dbe31a53",
      "blob": "5ba56e0dbb7cc433d5b98ab3cbbf5e0d92954ff1"
    }
  }
}